Sprint 08 - minor changes

DanteIT94 opened the pull request.

DanteIT94 committed:
Sprint 08 - minor changes

DanteIT94 committed:
Sprint 8 - Commit # 2

1) Logo on LaunchScreen was optimized according to Figma
2) LikeButton was reworked according to Figma sizes
3) Fonts are corrected
4) Logic of LikeButton (white and red hearts) was changed - now its working ))

5) Status bar now is white
6) Background of Cell is LightGrey now (because it's important to see tap-action)
7) Was made GradientLayer on the bottom of Cell

DanteIT94 committed:
Sprint 8 - Commit # 3

1) appDelegate file - "Sleep(3)" was deleted
2) was fixed sizes of Like - button (it was missunderstanding)

DanteIT94 committed:
Sprint 8 - minor fixes

Helvetii951 commented on pinterestClone/Base.lproj/Main.storyboard:
```diff
@@ -44,24 +44,22 @@
                                                 </label>
-                                                <button opaque="NO" contentMode="scaleToFill" misplaced="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="htv-lr-XWR">
-                                                    <rect key="frame" x="318.33333333333331" y="4" width="21" height="18"/>
+                                                <button opaque="NO" contentMode="scaleToFill" fixedFrame="YES" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="htv-lr-XWR">
```
Кнопка до сих пор не на месте. Тебе нужно к кнопке добавить top и leading constraint к top и leading картинки, constant = 0. Также тебе нужно будет добавить constraint высоты и ширины 42x42.

Должно получится то-то типа того:

https://ibb.co/p2pQgzS

DanteIT94 committed:
Update ImageListViewController.swift

DanteIT94 committed:
Sprint 08 - Commit # 4 (LikeButton Changes)

Finally I've added constraits of likeButton (42 x 42).
Now its working similarly on IPHONE X, SE, 8, 13 mini, 14 Pro and others

DanteIT94 committed:
Sprint 8 - Commit # 5 (Minor fixes)

1) Was renamed image of Like and Unliked Buttons
2) Was added constrateds and fixes sizes
3) Was added 2x and 3x images of likeButtons
